manifests/examples/azure/jessie.yml

```diff
@@ -23,7 +23,11 @@ volume:
     root:
       filesystem: ext4
       size: 10GiB
-packages: {}
+packages:
+  components:
+  - main
+  - contrib
+  - non-free
```

Comment:
Same here. contrib + non-free are not added per default. Is there some specific motivation for deviating from the default config?